pom.xml

```diff
@@ -77,7 +77,10 @@
     <spotbugs.threshold>${findbugs.threshold}</spotbugs.threshold>
     <!-- Defines a SpotBugs effort. Use "Max" to maximize the scan depth -->
     <spotbugs.effort>${findbugs.effort}</spotbugs.effort>
-    
+
+    <!-- Version of slf4j, only used when BOM is not used, but needs to be at the top level to be usable by plugins -->
```

Comment:
it is not supposed to be usable by plugins, so this comment is wrong. 
the dependencies for slf4j are in `dependencyManagement` or the `bom` so you *should* omit the version entirely.

sadly the flatten maven plugin can not cope with this, but the comment should really be addressed that this should not be used and is only a workaround for plugins producing incrementals and also declaring a dependency on and slf4j artifact.

Reply:
OK, I've rephrased the comment. I hope I got it right.

I don't insist on this change. Stash Pull Request Builder plugin can use plugin-pom 3.53 for now and switch to BOM once it becomes more reasonable to require a newer Jenkins.

Reply:
Speaking of the newer Jenkins - `README.md` is this repository implies that Jenkins 2.195 should be required, which is newer than the latest stable Jenkins version - way too new for plugin developers.
However, the `bom` package provides retroactive dependencies for stable branches starting with 2.138.x and its `README.md` recommends setting `jenkins.version` to 2.138.4, which is entirely reasonable.
`bitbucket-branch-source-plugin` has already switched to BOM and it requires Jenkins 2.138.4. I'm going to explore a similar change in Stash Pull Request Builder.
I think `README.md` in this repository should be clarified.

Reply:
jenkins-bom and plugin-bom is not the same @proski

This is mostly for plugins: https://github.com/jenkinsci/bom
This is jenkins bom: https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml

Reply:
!!Do not use the BOM profile!!

Reply:
@jtnord too late? plenty of people use it.
https://github.com/search?q=org%3Ajenkinsci+io.jenkins.tools.bom+extension%3Axml&type=Code

Reply:
wrong bom I'm talking about the [profile](https://github.com/jenkinsci/plugin-pom/blob/237da957dbad02bcb084e6c35aa505eff3654f5b/pom.xml#L920)   so more bom confusion :)